.github/workflows/events-recurrent.yml

```diff
@@ -1,43 +0,0 @@
-name: Recurrent
-
-# https://jasonet.co/posts/scheduled-actions/
-# https://github.community/t/distinct-job-for-each-schedule/17811/2
-on:
-  push:
-    branches: [ master ]
-  schedule:
-    - cron: "*/20 * * * *" # At every 20 minutes
-
-env:
-  GKE_CLUSTER: lightning-cluster
-  GKE_ZONE: us-central1-a
-
-jobs:
-  tpu-cleanup:
-    name: TPU cleaning
```

Comment:
I don't understand why we had this if the circleci and kubernetes job had timeouts